install/scripts/fcrepo.sh

```diff
@@ -30,4 +30,6 @@ service tomcat7 restart
 
 sleep 10
 cp -v $HOME_DIR/islandora/install/configs/repository.json /var/lib/tomcat7/webapps/fcrepo/WEB-INF/classes/config/minimal-default/repository.json
+echo "CATALINA_OPTS=\"\${CATALINA_OPTS} -Dfcrepo.modeshape.configuration=CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.modeshape.configuration=classpath:/config/minimal-default/repository.json"
```

Comment:
@ruebot Doh! @Natkeeran's problem is because this config no longer exists. Modeshape config here needs to be like https://github.com/fcrepo4-exts/fcrepo4-vagrant/commit/05fcddc2e675ca0fc978fbfd9c3fb44ce111d478